src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Serialization/ObjectWriter.cs

```diff
@@ -74,6 +76,8 @@ private static class BufferPool<T>
     /// </summary>
     private WriterReferenceMap _stringReferenceMap;
 
+    private static readonly Encoding s_encoding = Encoding.UTF8;
```

Comment:
Why make this a static readonly? I believe it should already be cached on both Framework and modern. If the goal is just to avoid specifying `Encoding.UTF8` in multiple places, consider making this a property instead, but I'm personally fine with just using UTF8 directly in both locations.

Reply:
Sure, I've gone ahead and changed